recipes-debian/insserv/insserv_debian.bb

```diff
@@ -0,0 +1,52 @@
+# base recipe: meta-debian/insserv/insserv_debian.bb
+# base branch: warrior
+# base commit: 3f609957cd1c373ced7ad32d21e5d688f118eba0
+SUMMARY = "Boot sequence organizer using LSB init.d dependencies"
+DESCRIPTION = "This utility reorders the init.d boot scripts based on \
+dependencies agiven in scripts'LSB comment headers, or in override files \
+included in this package or added in /etc/insserv."
+# There is no known home page for insserv
+
+LICENSE = "GPLv2"
+LIC_FILES_CHKSUM = "file://COPYING;md5=751419260aa954499f7abaabaa882bbe"
+
+inherit debian-package
+require recipes-debian/sources/insserv.inc
+
+#Add option -D-GNU_SOURCE: The O_NOATIME variable in insserv.c \
+#is depends on __USE_GNU and __USE_GNU depends on _GNU_SOURCE
+CFLAGS += " -D_GNU_SOURCE"
+
+S = "${WORKDIR}/${BPN}"
```

Comment:
If DEBIAN_UNPACK_DIR has already set,  [deibian-package.bb](https://github.com/meta-debian/meta-debian/blob/warrior/classes/debian-package.bbclass#L11) class sest DEBIAN_UNPACK_DIR to S variable.
so,  define ``` DEBIAN_UNPACK_DIR = "${WORKDIR}/${BPN}" ``` is enough.

Reply:
Corrected as indicated.